### PR TITLE
[Elm] Add missing operation summary

### PR DESCRIPTION
**Pull request by Qluxzz**
We currently only print the notes in a doc comment if they exist. This PR fixes so both the summary and notes are printed as expected.

### PR checklist
 
- [X] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [X] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [X] Run the following to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) and update samples:
  ```
  ./mvnw clean package || exit
  ./bin/generate-samples.sh ./bin/configs/*.yaml || exit
  ./bin/utils/export_docs_generators.sh || exit
  ``` 
  (For Windows users, please run the script in [Git BASH](https://gitforwindows.org/))
  Commit all changed files. 
  This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. 
  These must match the expectations made by your contribution. 
  You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. 
  IMPORTANT: Do **NOT** purge/delete any folders/files (e.g. tests) when regenerating the samples as manually written tests may be removed.
- [X] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master` (upcoming `7.x.0` minor release - breaking changes with fallbacks), `8.0.x` (breaking changes without fallbacks)
- [X] If your PR is targeting a particular programming language, @mention the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) members, so they are more likely to review the pull request.

### CODE REVIEW

**Comment by wing328** [comment]
thanks for the PR

cc @eriktim

**Review by eriktim — CHANGES_REQUESTED** [review]

File: modules/openapi-generator/src/main/resources/elm/operation.mustache [hunk 1]
```diff
@@ -30,10 +30,9 @@ import File exposing (File){{/includeFile}}
 {{#operations}}
 {{#operation}}
 
-{{#notes}}
-{-| {{{notes}}}
+{{#summary}}{-| {{{summary}}}{{#notes}}. {{{notes}}}{{/notes}}
```
[the remark this answers is not in the document]
Comment (Qluxzz) [reply modules/openapi-generator/src/main/resources/elm/operation.mustache:33]: I tried adding them on separate lines first, but the Elm extension for VS Code then treats the subsequent lines as code blocks. So according to the https://package.elm-lang.org/help/documentation-format it should be written on a single line to have the correct formatting.

I don't know the semantics of the notes, if they should be considered code blocks or just an extension to the summary. The examples in pet store makes it seem like the should be an extension to the summary and not 

Here's an example on how it looks when using newlines
image

[the remark this answers is not in the document]
Comment (eriktim) [reply modules/openapi-generator/src/main/resources/elm/operation.mustache:33]: image

this looks like Markdown to me, `    ` (4 spaces) makes the text a code block.

[the remark this answers is not in the document]
Comment (Qluxzz) [reply modules/openapi-generator/src/main/resources/elm/operation.mustache]: You're correct, I have made the notes now not have any indentation and they are now not marked as code blocks anymore. Thanks for investigating.

**Review by eriktim — APPROVED** [review]
LGTM, thanks!

**Pull request merged**